news/2020-09-05-mappers-report-august.md

```diff
@@ -0,0 +1,195 @@
+- **osu!taiko:** [Ulqui](https://osu.ppy.sh/users/1263669) & [Jaltzu](https://osu.ppy.sh/users/2597417)
+- **osu!catch:** [Rocma](https://osu.ppy.sh/users/566276), [JierYagtama](https://osu.ppy.sh/users/7483452) & [autofanboy](https://osu.ppy.sh/users/636114)
+- **osu!mania:** [Murumoo](https://osu.ppy.sh/users/8001433), [RiP46](https://osu.ppy.sh/users/5222596) & [Stelar](https://osu.ppy.sh/users/15205070)
+
+Do you want to see your own name on this list? Do you like modding and helping other people with their beatmaps? If your answer to both of these questions is "yes", then you can also become a Beatmap Nominator! The applications are always open. 
+
+Head over to the [Beatmap Nominator website](https://bn.mappersguild.com) and apply today!
+
+### BN Departures
+
+- **osu!taiko:** [Yales](https://osu.ppy.sh/users/2377881) & [Axer](https://osu.ppy.sh/users/7299864)
+- **osu!catch:** [Dako](https://osu.ppy.sh/users/11081858)
+- **osu!mania:** [Kyousukee](https://osu.ppy.sh/users/8842107)
+
+## <a id="mappers-showcase"></a>Mappers' Showcase
+
+It's time again for the Mappers' Showcase! As every month we would like to present you some lesser-known mappers which are doing great content! 
+As we finished the first cycle with all modes and the Storyboarder showcase, we're going back to the beginning, starting again with the osu! game mode.
+
+Thanks to [Pachiru](https://osu.ppy.sh/users/2850983) and [hypercyte](https://osu.ppy.sh/users/9155377) for the selections of this month.
+
+### Scyhei
+
+Hidden in the shadow, [Scyhei](https://osu.ppy.sh/users/10772337) have been in the mapping scene for a few years already and have been constantly improving and innovating their mapping over the years. Despite having a lot of unfinished or unpublished sets, their mapping mastery in the wide panel of song-style and mapping style is what gives them a lot of potential to stand out in the mapping scene. 
+
+They proved their ability to map without high-slider velocities or high difficulties, and juggle between voice and background beats with their maps of [School Food Punishment - in bloom](https://osu.ppy.sh/beatmapsets/1248994) or [Sonoko Inoue - Kotonoha no Omoi](https://osu.ppy.sh/beatmapsets/993109) that remains fun and consistent during the whole map length. Their expertise don't stop here, they also handle rhythm exploitation at high BPM perfectly with their [Suzuki Konomi 'n Kiba of Akiba - Watashi ga Motenai no wa Dou Kangaete mo Omaera ga Warui](https://osu.ppy.sh/beatmapsets/1249388) or [MEMAI SIREN - image \_\_\_\_\_](https://osu.ppy.sh/beatmapsets/1174170).
+
+If you enjoy clean beatmaps structure, a balanced amount of circles and sliders with slight slider curves, Scyhei's maps are definitely made for you!
```

Comment:
- beatmaps should be singular
- place an and after structure since it's the end of the "list"